src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -406,11 +393,7 @@ const formConfig = {
             get('sponsorIsDeceased', formData) &&
             get('sponsorDeathConditions', formData),
           CustomPage: FileFieldCustom,
-          CustomPageReview: props =>
-            FileViewField(
-              props?.data.sponsorCasualtyReport || [],
-              `${sponsorWording(props?.data)} casualty report`,
-            ),
+          CustomPageReview: null,
```

Comment:
Replaced all `CustomPageReview`s that were for file uploads with `null` so that the regular `CustomPage` would be used in both locations.